tests/subscriptions.test.ts

```diff
@@ -0,0 +1,149 @@
+import { createTestSchema, runPlugin } from "./utils/test-helpers";
+
+describe("GraphQL Subscriptions", () => {
```

Comment:
thought: not sure we even need to support this, i dont think we've ever actually used subscriptions